src/EnergyPlus/BaseboardElectric.cc

```diff
@@ -509,7 +508,7 @@ namespace BaseboardElectric {
             CompName = baseboard->Baseboard(BaseboardNum).EquipName;
             state.dataSize->DataFracOfAutosizedHeatingCapacity = 1.0;
             state.dataSize->DataZoneNumber = baseboard->Baseboard(BaseboardNum).ZonePtr;
-            SizingMethod = HeatingCapacitySizing;
+            SizingMethod = static_cast<int>(AutoSizingType::HeatingCapacitySizing);
```

Comment:
The `SizingMethod` variable itself should probably be an `enum` and not an `int`.

Reply:
Yes, that was coming for sure, and since it is used for indexing, it will be seamless once we change to regular enums.